images/index.js

```diff
@@ -54,7 +55,7 @@ const getSettings = async () => {
     {
       name: 'individualAssets',
       alias: 'i',
-      type: (value) => ([0, 1].includes(Number(value)) ? Number(value) : 0),
+      type: (value) => ([0, 1].includes(Number(value)) ? Number(value) : 1),
```

Comment:
A more succinct notation for converting to a number is '+':

type: (value) => ([0, 1].includes(+value) ? +value : 1),

Reply:
changed!